### ISSUE TITLE
distance_array of (MetricType.IP, IndexType.IVF_PQ) is ascending order?

### ISSUE THREAD

**Issue by cslin0915** [milvus-io/milvus#2801]
**Please state you issue using the following template and, most importantly, in English.**

**Describe the bug**
The distances returned by search is ascending order, but it doesn't seem correct for the metric of inner product. 

**Steps/Code to reproduce behavior**
```python
param = {
    'collection_name': 'example_collection_',
    'dimension': 512,
    'index_file_size': 32,  
    'metric_type': MetricType.IP  
}
random.seed(0)
vectors = [[random.random() for _ in range(512)] for _ in range(10000)]
vectors = sklearn.preprocessing.normalize(vectors)
vectors = vectors.tolist()
status, ids = milvus.insert(
    collection_name='example_collection_', records=vectors, ids=list(range(10000))
)
index_param = {
    'nlist': 1024,
    'm': 16
}
milvus.create_index('example_collection_', IndexType.IVF_PQ, index_param)
query_vectors = vectors[0:10]
search_param = {
    "nprobe": 16
}
param = {
    'collection_name': 'example_collection_',
    'query_records': query_vectors,
    'top_k': 10,
    'params': search_param,
}
status, results = milvus.search(**param)
```

**Expected behavior**
- The distances should be descending order.

**Environment details**
- Method of installation : Docker
- Milvus version : milvusdb/milvus:cpu-latest
- Milvus configuration : default

**Snapshots**

```plain
distance_array : [[0.16953793168067932, 0.2836434245109558, 0.2866434156894684, 0.2873148024082184, 0.2874112129211426, 0.29034480452537537, 0.29188400506973267, 0.2919117212295532, 0.2921820282936096, 0.29240378737449646], ...]
```

**Comment by del-zhenwu** [comment]
```
import random
import sklearn.preprocessing
from milvus import Milvus, IndexType, MetricType


milvus = Milvus(host="127.0.0.1", port=19532)
param = {
    'collection_name': 'example_collection_',
    'dimension': 512,
    'index_file_size': 32,  
    'metric_type': MetricType.IP  
}
milvus.create_collection(param)
random.seed(0)
vectors = [[random.random() for _ in range(512)] for _ in range(10000)]
vectors = sklearn.preprocessing.normalize(vectors)
vectors = vectors.tolist()
status, ids = milvus.insert(
    collection_name='example_collection_', records=vectors, ids=list(range(10000))
)
index_param = {
    'nlist': 1024,
    'm': 16
}
milvus.create_index('example_collection_', IndexType.IVF_PQ, index_param)
query_vectors = vectors[0:10]
search_param = {
    "nprobe": 16
}
param = {
    'collection_name': 'example_collection_',
    'query_records': query_vectors,
    'top_k': 10,
    'params': search_param,
}
status, results = milvus.search(**param)
print(results)
```

**Comment by shengjun1985** [comment]
I am so sorry that the distance of IVF_PQ IP is wrong In 0.10.0.
And we have fixed it this week.
Please wait for 0.10.1.

**Comment by shengjun1985** [comment]
Fixed in pr #2784
